modules/docs/arrow-docs/README.md

```diff
@@ -93,22 +93,28 @@ We use Jekyll so you can deploy your docs to a local server to test your changes
 
 So, after making you doc changes as mentioned before, you would:
 
-### 1. Run Ank to get your docs deployed locally
+### 1. Run Dokka and Ank to get your docs deployed locally
 
-Ank is in charge of compiling and validating your doc snippets and deploying the proper binaries for those. Run the following command:
+Dokka is responsible of generating documentation based on source code annotations, while Ank is in charge of compiling and validating your doc snippets and deploying the proper binaries for those. Run the following commands in Arrow root dir:
```

Comment:
Can we be explicit about the ordering? It's important to note that you must run dokka first so Ank can copy over all the generated files afterwards.
I'd personally also highlight the "in Arrow root dir" with bold style or something.

Reply:
I’ve addressed your comments, thanks @JorgeCastilloPrz